atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -119,6 +120,21 @@ public boolean useTableMapping() {
     @JsonProperty("forceTableMappingIAmOnThePersistenceTeamAndKnowWhatIAmDoing")
     public abstract Optional<Boolean> forceTableMapping();
 
+    /**
+     * This should only be used in the case that metadata or the underlying schema in Oracle mismatch for a table.
+     * Adding a table reference here will perform either an alter, or to allow the metadata to be updated on startup.
+     *
+     * Generally speaking the operation is safe to perform, although it's on the operator to determine what the side
+     * effects are. For example, if the issue arose as two services are configured to use this table, but only one is
+     * performing table mapping, then it is expected that this could break one of the services. However, that condition
+     * still satisfies the status quo, thus it's on the configurator to determine if this change is safe to make.
+     */
+    @JsonProperty("alterTablesOrMetadataToMatchOnStartupAndIKnowWhatIAmDoing")
+    @Value.Default
+    public List<TableReference> alterTablesOrMetadataToMatch() {
+        return List.of();
```

Comment:
nit: we don't actually need Default here, since lists default to empty in Immutables I think.

Reply:
oh duh!